tests/test_sanity.py

```diff
@@ -23,64 +23,76 @@
 def test_java_from_host(
     tmp_path: Path,
     application_pid: int,
-    assert_collapsed: Callable[[Optional[Mapping[str, int]]], None],
+    assert_collapsed,
 ) -> None:
     with JavaProfiler(1000, 1, Event(), str(tmp_path)) as profiler:
-        process_collapsed = profiler.snapshot()
-        assert_collapsed(process_collapsed.get(application_pid))
+        process_collapsed = profiler.snapshot().get(application_pid)
+        assert_collapsed(process_collapsed, True)
```

Comment:
```suggestion
        assert_collapsed(process_collapsed, check_comm=True)
```
please be explicit here since `assert_collapsed` is partial without full signature..
In all occurences

Reply:
k